ctapipe_io_lst/calibration.py

```diff
@@ -359,40 +351,38 @@ def interpolate_pseudo_pulses(waveform, expected_pixel_id, fc, fc_old, n_modules
         n_modules : int
             Number of modules
         """
-        roi_size = 40
-        size1drs = 1024
-        size4drs = 4096
-        n_gain = 2
-        n_pix = 7
-        for nr_module in prange(0, n_modules):
```

Comment:
Just for curiosity: `@njit()` instead `prange` give similar speed of execution?

Reply:
One has nothing to do with the other. `njit` is a short form for `jit(nopython=True)`, so it forces a successfull compilation or raises an error if compilation is not possible. `jit` will only warn and then use the slow pure python code if something goes wrong.

`prange` allows numba to parallelize the loop to multiple CPU cores. I removed it for debugging and will put it back if it provides a benefit. However, nesting `pranges` is certanly not a good idea. Only the outermost loop over the pixels probably makes sense as `prange`.